### PR TITLE
Remove before/after suite

### PR DESCRIPTION
**Pull request by adityagesh**
before_suite and after_suite was deprecated in
PR #2662 with 3 month waiting period

### CODE REVIEW

File: lisa/testsuite.py
Comment (squirrelsc) [inline lisa/testsuite.py:597]: I think we should add one more step, to raise an error, so it's obvious to people to fix.

File: lisa/testsuite.py
Comment (squirrelsc) [inline lisa/testsuite.py]: The error message is not friendly, because it's not actionable. Reuse existing error messages.

**Review by squirrelsc — APPROVED** [review]

**Comment by LiliDeng** [comment]
@adityagesh @squirrelsc don't merge it, I saw other teams are using it. Let's do the communication before merging.

**Pull request merged**
